consensus/hotstuff/votecollector/consensus_cluster_vote_collector.go

```diff
@@ -0,0 +1,120 @@
+package votecollector
+
+import (
+	"fmt"
+
+	"go.uber.org/atomic"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type ConsensusClusterVoteCollector struct {
+	BaseVoteCollector
+
+	dkg           hotstuff.DKG
+	aggregator    CombinedAggregator
```

Comment:
are we using the combined aggregator or [the splitted stakingSigAggregator and thresholdSigAggregator](https://github.com/onflow/flow-go/pull/1089/files#diff-abe94730661c5ebfd554104b47f9c67372cdca09c5ae95ad64bed79091790a2cR11-R12)?

Reply:
I thought that we will use Combined aggregator for staking + threshold sig and reconstructor for random beacon